espn_api/base_league.py

```diff
@@ -120,6 +120,23 @@ def _get_all_pro_schedule(self):
             pro_team_schedule[team['id']] = pro_game
         return pro_team_schedule
 
+    def _get_offers(self, week: int = None):
+        '''Returns a list of free agent auction bids'''
+        if week is None:
+            bids = []
+            for week in range(0, 18):

```

Comment:
For base league this could be used for other sports, I would suggest using the field `self.finalScoringPeriod` for rnage